lib/process-services/src/lib/task-list/components/task-details.component.scss

```diff
@@ -70,6 +71,7 @@ adf-task-header.adf-assign-edit-view ::ng-deep adf-card-view ::ng-deep .adf-prop
 
         &-form {
             flex-grow: 1;
+            margin-bottom: 8px;
 
             & ::ng-deep .adf-form-debug-container {
```

Comment:
this is another problem, not directly related to this PR but could be nice to fix, we should not have the old "::ng-deep" in the styles

Reply:
that's the symptom of missing ViewEncapsulation setup

Reply:
done